recipe/meta.yaml

```diff
@@ -245,11 +250,14 @@ outputs:
       run:
         - _libgcc_mutex {{ _libgcc_mutex }}
         - {{ pin_subpackage("libgomp", max_pin=None) }}
+      run_constrained:
+        # conflict with previous name
+        - openmp_impl 9999
```

Comment:
We could just mark all of the builds from my first PR as broken.

Reply:
Yes, but this is for setups with `openmp_impl` already installed.